backend.py

```diff
@@ -325,7 +326,9 @@ def __init__(self, board, robots):
         self._board = board
         self.robots = robots
         self.tile_count = self.get_tile_count()
-        self.new_card_pack = self.create_card_pack()
+        self.new_cards_pack = self.create_card_pack()
+        self.used_cards_pack = set()
+        self.game_register = 0
```

Comment:
Co to je ten atribut `game_register`? K čemu je?

Reply:
Pozor na terminologii. Hra má spoustu *kol*; v každém kole se zpracuje 5 *registrů*.

Reply:
To jsem z toho jelen. @befeleme  co to tedy v tomto případě to game_register? 
To bude ta jedna pětina vždy?

Reply:
Bude to herní kolo, až to opravím.